### DISCUSSION TITLE
Can I use pyodide as a kernel underneath jupyter_server?

### DISCUSSION THREAD

**Discussion by kdarby-cqg** [pyodide/pyodide#4843] [Q&A]
I know this is a strange question.  I want to set up a development environment outside of the browser using a pyodide python kernel and vscode (or vscode for web(!)).   Users would develop and do research in this environment, leveraging bigger cpus, and then deploy code to the production environment inside a browser. 

Just wanted to know if this was possible, and if not, if there were any hints about what it might take.

Thanks!
Kevin

**Comment by hoodmane** [comment]
Have you looked at jupyterlite?

**Reply by kdarby-cqg** [reply]
Yes, but I am under the impression that only runs in the browser, as far as I can tell.  Is there a way to say 'jupyter-lite server' ??

**Reply by hoodmane** [reply]
Why not ask there and cross reference this issue?

**Reply by kdarby-cqg** [reply]
Done, thanks: https://github.com/jupyterlite/jupyterlite/discussions/1407
